app/views/logs/index.html.erb

```diff
@@ -4,6 +4,18 @@
 <% content_for :title, title %>
 
 <% if current_page?(controller: 'lettings_logs', action: 'index') %>
+  <% if @logs_to_update > 0 %>
+    <%= govuk_notification_banner(
+      title_text: "Important",
+      title_heading_level: 3,
+      title_id: "impacted-logs-banner",
+    ) do |notification_banner| %>
+      <% notification_banner.heading(text: "A scheme has changed and it has affected #{@logs_to_update} #{@logs_to_update > 1 ? 'logs' : 'log'}") %>
+      <div class="govuk-notification-banner__heading">
+        <a class="govuk-notification-banner__link" href="./lettings-logs/update-logs">Update logs</a>
+      </div>
```

Comment:
Using explicit divs rather than the `heading_href` option on `notification_banner.heading` to force a newline, per Slack discussion with Nicky.